pages/reference/public-api/export-branch-stats.mdx

```diff
@@ -0,0 +1,59 @@
+{/* cSpell:disable */}
+
+# Export Branch stats
+
+_Export branch stats using Avo API_
+
+The Export API gives you the possibility to export your Avo branches as CSV.
+
+## Endpoint
+
+```Url
+https://api.avo.app/workspaces/:workspaceId/branches/export/v1
+```
+
+`:workspaceId` is the ID of your workspace. You'll find it in the URL of your avo tab after `/schemas/`.
+
+Returns [a CSV string that is a representation of your branch stats]
```

Comment:
Which branches are included in the response? All branches in history, only open ones, or something else?